contracts/cw-ics721-bridge/src/msg.rs

```diff
@@ -135,6 +135,55 @@ pub enum QueryMsg {
     /// Gets this contract's cw721-proxy if one is set.
     #[returns(Option<::cosmwasm_std::Addr>)]
     Proxy {},
+
+    /// Gets the code used for instantiating new cw721s.
+    #[returns(u64)]
+    Cw721CodeId {},
+
+    /// Gets a list of classID (from NonFungibleTokenPacketData) and
+    /// cw721 contract we have instantiated for that classID.
+    #[returns(Vec<ClassIdToNftContractResponse>)]
+    ClassIdToNftContract {
+        start_after: Option<ClassId>,
+        limit: Option<u32>,
+    },
+
+    /// Gets a list of class ID, token ID, and local channel ID. Used
+    /// to determine the local channel that NFTs have been sent
+    /// out on.
+    #[returns(Vec<ClassTokenToChannelResponse>)]
+    OutgoingClassTokenToChannel(ClassTokenToChannelQuery),
+
+    /// Gets a list of class ID, token ID, and local channel ID. Used
+    /// to determine the local channel that NFTs have arrived at
+    /// this contract.
+    #[returns(Vec<ClassTokenToChannelResponse>)]
+    IncomingClassTokenToChannel(ClassTokenToChannelQuery),
+}
+
+#[cw_serde]
+pub struct ClassTokenToChannelQuery {
+    pub start_after: Option<ClassToken>,
+    pub limit: Option<u32>,
+}
+
+#[cw_serde]
+pub struct ClassToken {
+    pub class_id: ClassId,
+    pub token_id: TokenId,
+}
+
+#[cw_serde]
+pub struct ClassIdToNftContractResponse {
+    pub class_id: String,
+    pub nft_contract: Addr,
+}
+
+#[cw_serde]
+pub struct ClassTokenToChannelResponse {
```

Comment:
A bit weird namings, the response is a Vec of `ClassIdToNftContractResponse`, and not the struct directly, if you remove the maps you can also remove those structs.